app/presenters/publishing_api/world_location_news_article_presenter.rb

```diff
@@ -60,6 +60,16 @@ def image_available?
       item.images.any?
     end
 
+    def prepend_slash(path)
+      path.start_with?("/") ? path : path.prepend("/")
+    end
+
+    def image_path
+      image_path = presented_world_location_news_article.lead_image_path
+
+      Whitehall.public_asset_host + prepend_slash(image_path)
```

Comment:
Using the [URI module](https://docs.ruby-lang.org/en/2.4.0/URI.html) not primitive strings would surface this by raising an exception if the URI is badly formed:

```rb
URI(Whitehall.public_asset_host).tap { |uri| uri.path = "placeholder.png" }
# => URI::InvalidComponentError: bad component(expected absolute path component): placeholder.png

URI(Whitehall.public_asset_host).tap { |uri| uri.path = "/placeholder.png" }.to_s
# => "http://assets-origin.dev.gov.uk/placeholder.png"
```